src/components/FloatingDropdown/FloatingDropdownBase.st.css

```diff
@@ -68,6 +68,14 @@
     width: 100%;
 }
 
+.displayBlockBtn {
+    width: 100%;
+}
+
+.displayBlockContent {
+    justify-content: space-between;
+}
+
```

Comment:
Really not critical, 
but I think I would have just added one class to the root component, and then use that to target the button and the content.
Something like:
```css
.displayBlock .btn {
  width: 100%;
}

.displayBlock .content {
  justify-content: space-between;
}
```

But again, this isn't critical